gen/build_deploy/bash/dcos-launch.spec

```diff
@@ -1,4 +1,12 @@
-a = Analysis(['test_util/launch.py'])
+# Warning:
+#   - hidden import must be used as there is a bug in pyinstaller
+#     https://github.com/pyinstaller/pyinstaller/issues/2185
+#   - data must be decalared explicitly if not a .py file
+#   - Building will suck up the local SSL .so and package it
+#     with the final exe. Ensure build system has OpenSSL 1.0.2g or greater
```

Comment:
This is only run by our CI / not run by people trying to build this themselves generally. In the case someone is doing local development and iterating on dcos-launch, then this pulling in whatever the local openssl is should be fine.